addons/atlassian-confluence/template/index.js

```diff
@@ -54,48 +59,48 @@ async function loadSpec (accessToken) {
         authorizations: {
             oAuthDefinitions: {
                 token: {
-                    access_token: accessToken
-                }
-            }
-        }
+                    access_token: accessToken,
+                },
+            },
+        },
     };
 
     // Parse the specification and generate the API.
-    return await new Promise((resolve, reject) => new SwaggerClient(swagSpec).then(client => resolve(client)));
-};
+    return await new Promise((resolve, reject) => new SwaggerClient(swagSpec).then((client) => resolve(client)));
+}
 
 async function listAccessibleResources(accessToken) {
     let response;
     try {
-        response = await Superagent("https://api.atlassian.com/oauth/token/accessible-resources")
-            .set("Authorization", "Bearer " + accessToken)
-            .set("Accept", "application/json");
+        response = await Superagent('https://api.atlassian.com/oauth/token/accessible-resources')
+            .set('Authorization', 'Bearer ' + accessToken)
+            .set('Accept', 'application/json');
     } catch (e) {
         console.log('Accessible Resources Exception');
         throw e;
     }
 
-// Example response:
-// [
-//   {
-//     "id": "d0a50dbb-9553-444e-80e3-5080fa25a3b4",
-//     "url": "https://magitech-test.atlassian.net",
-//     "name": "magitech-test",
-//     "scopes": [
-//       "search:confluence",
-//       "read:confluence-props",
-//       "read:confluence-space.summary",
-//       "read:confluence-content.summary",
-//       "read:confluence-content.all"
-//     ],
-//     "avatarUrl": "https://site-admin-avatar-cdn.prod.public.atl-paas.net/avatars/240/flag.png"
-//   }
-// ]
+    // Example response:
+    // [
+    //   {
+    //     "id": "d0a50dbb-9553-444e-80e3-5080fa25a3b4",
+    //     "url": "https://magitech-test.atlassian.net",
```

Comment:
can you excise out the 'magitech' keyword from this and the subsequent line, please?

Reply:
Fixed